uktrade/data-hub-frontend
# Move Typeahead into Datahub

## Description of change
 
(15) This is a small PR that moves the Typeahead files from Storybook over to DataHub. Tests have not been carried over yet (hence the poor codecov score), as we need to implement Jest into DH - however this is pending on another ticket. 

## Test instructions

Thanks to Dave, you can now run `npm run storybook` which should successfully bring up Storybook on your browser. navigate to the Typeahead component, and you should see everything working as expected. 
## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
